pkg/cmd/render/fixtures/cluster-api-cluster.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: Cluster
+metadata:
+  creationTimestamp: null
+  name: cluster
```

Comment:
this object may not be created here.  If the MCO needs this resource it must supply it's own render command or place it directly into the installer. The kube-apiserver-operator must not become a "this is a convenient place for other unrelated operators to place manifests".

Reply:
this is only for unit testing. The object is created by the installer. This component is consuming an object which needs to be removed see https://github.com/openshift/installer/pull/1449/files